common/jenkins-agents/maven/docker/set_maven_proxy.sh

```diff
@@ -22,16 +22,47 @@ mvn_proxy="<proxy><id>internal</id><active>true</active><protocol>http</protocol
 		proxy_pw=$(echo $proxy_userp | sed -e "s|$proxy_user:||g")
 		mvn_proxy=$mvn_proxy"<password>$proxy_pw</password>"
  	fi
-fi
 
-if [[ $NO_PROXY != "" ]]; then
-	noproxy_host=$(echo $NO_PROXY | sed -e 's|\,\.|\,\*\.|g')
-	noproxy_host=$(echo $noproxy_host | sed -e "s/,/|/g")
-	mvn_proxy=$mvn_proxy"<nonProxyHosts>$noproxy_host</nonProxyHosts>"
-fi
+ 	if [[ $NO_PROXY != "" ]]; then
+	 	noproxy_host=$(echo $NO_PROXY | sed -e 's|\,\.|\,\*\.|g')
+ 		noproxy_host=$(echo $noproxy_host | sed -e "s/,/|/g")
+ 		mvn_proxy=$mvn_proxy"<nonProxyHosts>$noproxy_host</nonProxyHosts>"
+ 	fi
 
-if [[ $HTTP_PROXY != "" ]]; then
 	mvn_proxy=$mvn_proxy"</proxy>"
 fi
 
+if [[ $HTTPS_PROXY != "" ]]; then
+
+mvn_proxy_s="<proxy><id>internal-https</id><active>true</active><protocol>https</protocol>"
```

Comment:
this changes the support to only `https`, shouldn't be compatible with also wiht `http`?. If not, I would change the feature title in `CHANGELOG.md` to reflect this.